src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -284,15 +286,16 @@ namespace winrt::TerminalApp::implementation
     void TerminalPage::_ShowCloseWarningDialog()
     {
         auto title = RS_(L"CloseWindowWarningTitle");
-        auto primaryButtonText = RS_(L"CloseAll");
-        auto secondaryButtonText = RS_(L"Cancel");
+        auto closeButtonText = RS_(L"CloseAll");
+        auto primaryButtonText = RS_(L"Cancel");
```

Comment:
this is still incorrect. The "close" button _is meant to exit the dialog_. The "primary" button is meant to _activate the dialog's action._.

Reply:
So the default button should be Cancel instead of CloseAll?

Reply:
primary=close all, close=cancel, default=primary.